src/core/connection.h

```diff
@@ -159,6 +159,13 @@ typedef union QUIC_CONNECTION_STATE {
         //
         BOOLEAN InlineApiExecution : 1;
 
+        //
+        // True once a client connection has completed a compatible version
+        // negotiation, and false otherwise. Used to prevent packets with invalid
+        // version fields from being accepted.
+        //
+        BOOLEAN CompatibleVersionNegotiation : 1;
```

Comment:
Please double check this doesn't push the size to 64-bits. If it does, you need to update the `uint32_t` flags to `uint64_t`.

Reply:
Good call! Had to upgrade to uint64_t.